wsps/export.index

```diff
@@ -83,3 +82,5 @@
     "828_ocspnavmenu": "828_OCSPNavMenu",
     "4_class_definitions": "4_Class-definitions"
 }
```

Comment:
Are you sure that you need two `}` here?

Reply:
I'm sure I don't. That must be an error in my merge. Weird, I thought for sure I discarded the lines above...
Anyway, it should be this:
`
    "4_class_definitions": "4_Class-definitions",
    "10_csp_searchbar": "10_Csp searchbar"
}
`

Reply:
Ugh... with linebreaks...